src/controllers/expense.controller.js

```diff
@@ -0,0 +1,94 @@
+const expenseService = require('../services/expense.service.js');
+const userService = require('../services/user.service.js');
+
+const get = async (req, res) => {
+  const { categories, userId, from, to } = req.query;
+
+  const result = await expenseService.getAll(categories, userId, from, to);
+
+  res.send(result);
+};
+
+const getById = async (req, res) => {
+  const { id } = req.params;
+
+  const expense = await expenseService.getById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = async (req, res) => {
+  const { userId, title, amount, category, note, spentAt } = req.body;
+  const existedUser = await userService.getById(userId);
+
+  if (!userId || !title || !amount || !spentAt || !existedUser) {
```

Comment:
The `create` function should validate that `amount` is a number. Consider adding a check like `typeof amount !== 'number'` to ensure the `amount` is valid.